core/templates/domain/suggestion/SuggestionThreadObjectFactory.ts

```diff
@@ -37,7 +37,9 @@ export class SuggestionThread {
   lastUpdatedMsecs: number;
   messageCount: number;
   threadId: string;
-  suggestion: Suggestion;
+  // A suggestion will be null when suggestionType is not equal
```

Comment:
This does not really tell what happens when the `suggestion` is `null`. Maybe explain this comment a little further?

Reply:
done